src/App.tsx

```diff
@@ -15,48 +16,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_LENGHT = 'length';
+const SORT_ALPHABETICALLY = 'alphabetically';
```

Comment:
Use enum SortType instead of string literals for possible sorting options (including the default one).

So, you have to rewrite it using enums :)